api-reference/50 Common/Object Structures/positionConfig/at/x.md

```diff
@@ -5,6 +5,6 @@ type: String
 ---
 ---
 ##### shortDescription
-Holds the horizontal position specifier.
+Specifies a position in the horizontal direction.
```

Comment:
Can we say:
Specifies the horizontal position.

Reply:
Okay, does it make sense to say something like:
Specifies a position in the horizontal direction (for left, right, or center alignment).